### PR TITLE
Fix documentation plugin tests

### PR DESCRIPTION
**Pull request by markkaylor**
### What does it do?

- Adds missing jest config to the documentation
- Fixes tests that were broken

### Why is it needed?

- The tests for the documentation plugin were not running

### How to test it?

From the root of the monorepo, run yarn 

```
yarn test:unit packages/plugins/documentation/server/services/__tests__
```
The tests should run and they should pass

### CODE REVIEW

File: packages/plugins/documentation/server/services/helpers/build-component-schema.js [hunk 1]
```diff
@@ -88,30 +88,61 @@ const getAllSchemasForContentType = ({ routeInfo, attributes, uniqueName }) => {
...
-              addComponentSchema,
+              didAddStrapiComponentsToSchemas,
             }),
           },
         },
       },
     };
   }
-
+  // Build the localization response schema
```
Comment (remidej) [inline packages/plugins/documentation/server/services/helpers/build-component-schema.js:98]: newline

File: packages/plugins/documentation/server/services/helpers/build-component-schema.js [hunk 2]
```diff
@@ -88,30 +88,61 @@ const getAllSchemasForContentType = ({ routeInfo, attributes, uniqueName }) => {
...
+            type: 'object',
+            properties: cleanSchemaAttributes(attributes, {
+              didAddStrapiComponentsToSchemas,
+            }),
+          },
         },
       },
     };
   }
-
```
Comment (remidej) [inline packages/plugins/documentation/server/services/helpers/build-component-schema.js:111]: I think the newline was nice

File: packages/plugins/documentation/server/services/helpers/build-component-schema.js [hunk 3]
```diff
@@ -203,7 +213,7 @@ const getAllSchemasForContentType = ({ routeInfo, attributes, uniqueName }) => {
       },
     },
   };
-  return { ...schemas, ...componentSchemas };
+  return { ...schemas };
```
Comment (remidej) [inline packages/plugins/documentation/server/services/helpers/build-component-schema.js:216]: newline before return

**Review by remidej — COMMENTED** [review]
Looks good to me, I just added a couple of formatting suggestions as the file seemed hard to digest

**Comment by markkaylor** [comment]
@remidej @Feranchz sorry could you review again I decided to go ahead and add a test to validate the generated schema matches the openapi specifcation

**Review by remidej — APPROVED** [review]
Checking the schema validity in the tests directly is a great idea 👍

**Pull request merged**